journal/modernized/02-chapter-1.adoc

```diff
@@ -837,12 +837,12 @@ where I arrived safely and staid some time.
 Afterwards I went into North Wales, having another Friend in the ministry along with me;
 and the Lord enabled us to proclaim his everlasting gospel,
 whereby many hearts were reached, and the churches of Christ confirmed.
-When we had visited North Wales, we travelled into the south parts; into Pembrokeshire,
+When we had visited North Wales, we travelled into the south parts into Pembrokeshire,
```

Comment:
This one's a little tricky, but I think ultimately we should probably put _back_ the semi-colon and then add a dash, for clarity.  I'm pretty sure that "into the south parts" means "the south parts of Wales", and then the next three names are parts of southern Wales, so "into the south parts" is kind of like a summary of the specifics that follow, and so needs some sort of punctuation, like in the original. The tricky thing is this is not how we would punctuate this nowadays, so without re-writing the sentence, it's a bit tough.  But I think for now, this would be better:

```ASCIIDOC
When we had visited North Wales, we travelled into the south parts;--into Pembrokeshire,
```